src/screen3d.ts

```diff
@@ -0,0 +1,120 @@
+import * as BABYLON from 'babylonjs';
+import backgroundMap from '../asset/kumpula_kerroskartat_2015_1.png';
+
+let beacon: BABYLON.Mesh;
+
+export const setPosition = (x: number, y: number): void => {
+  if (!beacon) {
+    return;
+  }
```

Comment:
I wonder if we could implement this so that the `beacon` variable is passed as an argument to `setPosition`. Now it looks like `setPosition` can be called before the beacon is initialized, and hence the `if (!beacon) { return; }` check right? If i'm reading this right, we could return the beacon from drawscreen3d function.

The advantage would be that then the global mutable variable is not needed.

Reply:
Thanks for you comments. I modified setPosition so that it now accepts the beacon as an argument. I also added a new method for creating beacons (addBeacon).